src/components/Loader.jsx

```diff
@@ -1,13 +1,19 @@
-import React from 'react';
 import PropTypes from 'prop-types';
 import { ClipLoader, BounceLoader } from 'react-spinners';
 import { Typography } from 'antd';
 import useSWR from 'swr';
-
+import React from 'react';
+import { useSelector } from 'react-redux';
 import fetchAPI from 'utils/http/fetchAPI';
+import { getBackendStatus } from 'redux/selectors';
 
 const { Text } = Typography;
-
+const DOWNLOAD_EXPERIMENT = 'download_experiment_s3';
```

Comment:
Nice job on removing the messages from here. I suggest you also return these codes from the worker. Why do you want them here?

Reply:
right, forgot to delete them